app/src/main/java/de/westnordost/streetcomplete/quests/max_weight/AddMaxWeightForm.kt

```diff
@@ -0,0 +1,175 @@
+package de.westnordost.streetcomplete.quests.max_weight
+
+import android.os.Bundle
+import androidx.appcompat.app.AlertDialog
+import android.text.method.DigitsKeyListener
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.AdapterView
+import android.widget.ArrayAdapter
+import android.widget.EditText
+import android.widget.Spinner
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.quests.AbstractQuestFormAnswerFragment
+import de.westnordost.streetcomplete.quests.InputUtil
+import de.westnordost.streetcomplete.quests.OtherAnswer
+import de.westnordost.streetcomplete.util.TextChangedWatcher
+
+import de.westnordost.streetcomplete.quests.max_weight.Measurement.*
+
+private enum class Measurement { METRIC, IMPERIAL }
+
+class AddMaxWeightForm : AbstractQuestFormAnswerFragment<MaxWeightAnswer>() {
+
+    override val otherAnswers = listOf(
+            OtherAnswer(R.string.quest_maxweight_answer_other_sign) { onUnsupportedSign() },
+            OtherAnswer(R.string.quest_maxweight_answer_sign_with_exceptions) { onUnsupportedSign() },
+            OtherAnswer(R.string.quest_maxweight_answer_noSign) { applyAnswer(NoMaxWeightSign("dummy")) }
+    )
+
+    private fun onUnsupportedSign() {
+        AlertDialog.Builder(activity!!)
+                .setMessage(R.string.quest_maxweight_unsupported_sign_request_photo)
+                .setPositiveButton(R.string.quest_generic_hasFeature_yes) { _, _ -> onClickCantSay() }
```

Comment:
Rather use the "ok" string. This string might be translated in certain languages as something like "Yes it has"